code/script.js

```diff
@@ -1,19 +1,205 @@
-// Start here
-
 // Step 1 - Welcome and introduction
-// Your code goes here
-alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
-)
+alert(`Welcome to our Virtual Pizzeria. Ready to order? - Click 'OK' to begin.`);
+
+//A statement which variable value will be the name the user enters.
+const name = prompt(`Please enter your name:`);
+
+//will log the users name in the console.
+console.log(`Customer: ${name}`);
```

Comment:
Please remove the the console.log() around the code. Generally they need to be deteled when the code has been tested and works as meant to 🧹